playbooks/prepare_host.yaml

```diff
@@ -106,3 +113,18 @@
       systemd:
         name: ceph-osd-losetup.service
         enabled: true
+        state: started
+    - name: Create volume group for ceph
+      lvg:
+        vg: vg_ceph
+        pvs: /dev/loop1
+    - name: Create ceph data LV
+      lvol:
+        vg: vg_ceph
+        lv: data
+        size: 96%VG
+    - name: Create ceph db LV
+      lvol:
+        vg: vg_ceph
+        lv: db
+        size: 4%VG
```

Comment:
I don't think this will work because some space is lost for metadatas, which is why I did `- 0.1`

Reply:
It seems to work 🤷‍♂️ I'd hold off changing it until it actually fails.

Reply:
ack